### PR TITLE
Fixed ollama knowledge example

### PR DESCRIPTION
**Pull request by vodkar**
According to #1459

Changes:

- Added `OllamaEmbedder` in `cookbook/providers/ollama_tools/knowledge.py` example, instead of default `OpenAIEmbedder`
- Added Documentation, to run an example

### CODE REVIEW

File: cookbook/providers/ollama_tools/knowledge.py [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
 
 db_url = "postgresql+psycopg://ai:ai@localhost:5532/ai"
 
 knowledge_base = PDFUrlKnowledgeBase(
     urls=["https://phi-public.s3.amazonaws.com/recipes/ThaiRecipes.pdf"],
-    vector_db=PgVector(table_name="recipes", db_url=db_url),
+    vector_db=PgVector(
+        table_name="recipes",
+        db_url=db_url,
+        embedder=OllamaEmbedder(host="http://localhost:11434"),
```
Comment (ysolanky) [inline cookbook/providers/ollama_tools/knowledge.py:28]: Please update to: `embedder=OllamaEmbedder(model="nomic-embed-text")`

Comment (vodkar) [reply cookbook/providers/ollama_tools/knowledge.py]: Nice change

I have updated this, but it also requires adding `dimensions=768` 

It works more stable now. What's the difference between those embedders?

**Review by manthanguptaa — APPROVED** [review]

File: cookbook/providers/ollama_tools/knowledge.py (hunk 1 above)
Comment (ysolanky) [reply cookbook/providers/ollama_tools/knowledge.py]: @vodkar please remove the host as it is not a necessary parameter for `OllamaEmbedder` class. `nomic-embed-text` is a highly efficient model to create embeddings locally.

Comment (vodkar) [reply cookbook/providers/ollama_tools/knowledge.py]: Removed

**Review by ysolanky — APPROVED** [review]

File: cookbook/providers/ollama_tools/knowledge.py (hunk 1 above)
Comment (moscarelli) [reply cookbook/providers/ollama_tools/knowledge.py]: This is not working, still geting message embbeder is not passed and the agent will use opeai embedder

**Pull request merged**